wgsl/index.bs

```diff
@@ -11416,7 +11416,17 @@ the following exceptions:
         of NaNs and infinities.
 * Implementations may ignore the sign of a zero.
     That is, a zero with a positive sign may behave like a zero a with a negative sign, and vice versa.
-* No rounding mode is specified.
+* No rounding mode is specified. An implementation may round a value up or down.
+    * In particular, overflow cases may round to infinity or to the nearest finite value:
+        * A positive intermediate value *x* outside the finite range of the floating point type
+            may be rounded up to &plus;&infin; or to the largest positive finite value for the type.
+        * A negative intermediate value *x* outside the finite range of the floating point type
+            may be rounded down to &minus;&infin; or to the largest negative finite value for the type.
+    * Note: If an infinity is produced, it may further be changed as described above.
+    * Note: An overflowing [=const-expression=] may cause a [=shader-creation error=],
```

Comment:
Are override expressions and constant expressions considered equivalent in this context, or should this also have language about operations for overrides leading to overflow causing pipeline creation errors?